buildout.cfg

```diff
@@ -77,7 +77,7 @@ input = inline:
     export PATH=${buildout:bin-directory}:$PATH
     cd webpack
     yarn
-    exec $SHELL
+    exec /bin/bash
```

Comment:
Is this to avoid problems if there are different shells defined as default?

Another question, not exactly related to this PR: do you have plans to add for example a webpack.cfg in https://github.com/collective/buildout.plonetest? You're starting to repeat some patterns in collective.newsticker and collective.upload.

Reply:
yes, this is to avoid problems with different shells.

I was thinking about it this week.. too much changed since this PR, now we use https://github.com/simplesconsultoria/sc.recipe.staticresources and I think this will be the right place to create somethink like plone KGS but for webpack dependencies.

Still need to find out if there is a good way to simplify and reuse ``webpack.cfg`` patterns, but for sure we can simplify ``packages.json`` dependencies versions.